client/kie-client/src/main/java/org/apache/servicecomb/kie/client/KieClient.java

```diff
@@ -0,0 +1,161 @@
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.servicecomb.kie.client;
+
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.apache.http.HttpStatus;
+import org.apache.servicecomb.kie.client.http.HttpResponse;
+import org.apache.servicecomb.kie.client.model.KVBody;
+import org.apache.servicecomb.kie.client.model.KVDoc;
+import org.apache.servicecomb.kie.client.model.KVResponse;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import javax.management.OperationsException;
+import java.io.IOException;
+import java.util.List;
+import java.util.Map;
+import java.util.Map.Entry;
+
+/**
+ * Created by   on 2019/10/28.
+ */
+public class KieClient {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(KieClient.class);
+
+    private KieRawClient httpClient;
+
+    public KieClient() {
+        this(new KieRawClient());
+    }
+
+    /**
+     * Customized host,port,domainName and if any one parameter is null, it will be defaults
+     *
+     * @param host
+     * @param port
+     * @param domainName
+     */
+    public KieClient(String host, int port, String domainName) {
+        this.httpClient = new KieRawClient.Builder().setHost(host).setPort(port).setDomainName(domainName).build();
+    }
+
+    public KieClient(KieRawClient serviceCenterRawClient) {
+        this.httpClient = serviceCenterRawClient;
+    }
+
+    /**
+     * Put create value of a key
+     *
+     * @param key
+     * @param kvBody
+     * @return
+     */
+    public String putKeyValue(String key, KVBody kvBody) {
+        try {
+            ObjectMapper mapper = new ObjectMapper();
+            HttpResponse response = httpClient.putHttpRequest("/kie/kv/" + key, null, mapper.writeValueAsString(kvBody));
+            if (response.getStatusCode() == HttpStatus.SC_OK) {
+                LOGGER.info("putKeyValue result:" + response.getContent());
+                return response.getContent();
+            } else {
+                throw new OperationsException(response.getStatusCode() + response.getMessage() + response.getContent());
+            }
+        } catch (IOException | OperationsException e) {
+            e.printStackTrace();
+        }
+        return null;
+    }
+
+    /**
+     * Get get value of a key
+     *
+     * @param key
+     * @return
+     */
+    public List<KVResponse> getValueOfKey(String key) {
```

Comment:
Since this operation will not throw exception, should document this API that when return null, it is some error happens. And maybe define some XXXException for this module is better for users to use this API.